### PR TITLE
Charting

### LINKED ISSUE #203: Charting

### PR DESCRIPTION
**Pull request by Birdbh**
Fixes #203

### CODE REVIEW

File: src/flaskr/templates/main/home.html [hunk 1]
```diff
@@ -16,30 +18,40 @@ <h1>{% block title %}Home Page | Operations Optimization{% endblock %}</h1>
...
-    <option value="option3" class="dropdown-option">Both</option>
-  </select>
-</div>
-<br>
-{% endblock %}
+  <div class="box">
+    <select class="dropdown-list">
+      <option value="option1" class="dropdown-option">Clear Outliers</option>
+      <option value="option2" class="dropdown-option">Smooth</option>
+      <option value="option3" class="dropdown-option">Both</option>
```
Comment (Jcp149) [inline src/flaskr/templates/main/home.html:26]: there is no "smooth" there should be clear outliers, remove min and remove max

File: src/flaskr/templates/main/home.html [hunk 2]
```diff
@@ -16,30 +18,40 @@ <h1>{% block title %}Home Page | Operations Optimization{% endblock %}</h1>
...
+  </div>
+  <br>
 
-{% block content %}
-Upload Data:
-<form action="/upload" method="post" enctype="multipart/form-data">
-    <input type="file" name="file" id="file">
-    <input type="submit" value="Upload">
-</form>
+  <form action="/upload" method="post" enctype="multipart/form-data">
```
Comment (Jcp149) [inline src/flaskr/templates/main/home.html:31]: should this only be for premium users?

**Comment by Birdbh** [comment]
Additional fields in html have been removed to be added later on a future pull request, this is only for charting functionality. Home re-direct has also been fixed.

File: src/flaskr/templates/base.html
Comment (brkykrclk) [inline src/flaskr/templates/base.html:1]: So I already did this change for home.py, should I just ignore it?

File: src/flaskr/templates/main/home.html [hunk 3]
```diff
@@ -16,30 +18,29 @@ <h1>{% block title %}Home Page | Operations Optimization{% endblock %}</h1>
...
+          borderColor: 'rgba(255, 99, 132, 1)', // Color of the line
+          fill: false, // Specifies that the area under the line should not be filled
+          borderWidth: 1 // Width of the line
+        }]
+      },
+      options: {
+        responsive: false // Chart will not resize automatically when the container does
+      }
+    });
+  </script>
```
Comment (brkykrclk) [inline src/flaskr/templates/main/home.html:45]: I can also update home.py according to the new HTML file

**Review by brkykrclk — APPROVED** [review]
Looks great :D and please let me know about updates so I can send my pr too

**Pull request closed without merging**